### DISCUSSION TITLE
调试log如何查看？

### DISCUSSION THREAD

**Discussion by Geek-Joe** [SagerNet/SagerNet#160] [Q&A] [answered]
我自定义的配置文件（config.json），碰到一点问题需要调试。我到设置里面使能了log，也到配置里面置为了debug级别的调试输出，但是到哪里查看输出的日志？（v2rayNG有个logcat）

**Comment by nekohasekai** [comment, marked as answer]
关于 - 导出调试信息

**Reply by Geek-Joe** [reply]
1. 已经搞定，感谢
2. 发现用SagerNet后网速比V2rayNG快多了！体验很棒！
3. SagerNet也是v2ray-core最新内核，这样我在PC上的规则可以直接无修改放到SagerNet里用，不用维护2份了
4. 小小的打赏了下，请作者喝杯奶茶以表感谢
